### PR TITLE
media-libs/opencv: Fix contrib tarball name (PN -> P)

### PR DESCRIPTION
**Pull request by a17r**
This revbump might lead to new breakage since up to now opencv-3.1.0 had been using some >=3.0.0 contrib snapshot. At least it was discovered by some package _not_ being broken with opencv-3.1.0 as expected. ;)

### CODE REVIEW

**Comment by monsieurp** [comment]
@akhuettel @gentoo/proxy-maint

**Comment by gktrk** [comment]
@monsieurp can you also mention @Ankwyell? github doesn't let me do that.

**Comment by Ankwyell** [comment]
Don't see point of his pull req.

**Comment by a17r** [comment]
The current 3.1.0 opencv ebuild picked up the same unversioned 'opencv_contrib.zip' tarball from the mirrors as 3.0.0, while there is a proper 3.1.0-contrib release. There might be a reason for this, though, that I am not aware of?

**Comment by gktrk** [comment]
Hi, this PR removes the following patches: 3.1.0-fix-hal-headers.patch 3.1.0-fix-ocl-declaration.patch 3.1.0-invalid-string-usage.patch. I don't see how renaming a tarball achieves this. It is also not mentioned in the commit message.

**Comment by a17r** [comment]
It's a _different_ tarball already containing these fixes. But yeah they are removed in the wrong commit, I'm going to fix this.

**Comment by Ankwyell** [comment]
Ah now I can see it. It's a problem with distfiles. Thanks let me check it. Then I will merge it.

**Comment by Ankwyell** [comment]
I merged the fix for the contrib name. So I can merge also your "drop old" but I guess it would be better to say something about removing patches :)

**Comment by idella** [comment]
The commit df4c410 clearly states this is a revbump
The commit msg ought read similar to:
media-libs/opencv: revbump to vn. 3.1.0-r2
blank line 2
Explanatory text of revbump
Fix contrib tarball name (PN -> P) might just do it but it 's quite oblique.
I suggest you re-write it to something simpler and more descriptive. I am guessing PN -> P actually means ${PN} -> ${P} which even then is a coded abbreviation.

Also I am quite confused about "I merged the fix for the contrib name."
A pull request is merged as a whole so what was merged here I cannot guess since it cannot, or should not, be one of a total collection of git patches of a single pull request.

" your "drop old" but I guess it would be better to say something about removing patches :)"
is valid since it urging you to make a clear statement in a commit msg re removal of many defunct patches.

In short, the commit msg need be re-done from scratch

**Comment by Amynka** [comment]
done
